x/genutil/server/cli/gentx.go

```diff
@@ -1,9 +1,8 @@
-package init
```

Comment:
Same as previous comment on directory name.

Reply:
let's keep conversation in that other comment